toolkit/Makefile

```diff
@@ -48,6 +48,7 @@ INITRD_CACHE_SUMMARY            ?=
 PACKAGE_ARCHIVE                 ?=
 PACKAGE_BUILD_RETRIES           ?= 1
 CHECK_BUILD_RETRIES             ?= 1
+EXTRA_BUILD_LAYERS              ?= 0
```

Comment:
Since we're here and it's related, I think it's time to remove `REBUILD_DEP_CHAINS` both from this file and the `building.md` one.